src/main.js

```diff
@@ -42,6 +43,15 @@ __webpack_nonce__ = btoa(getRequestToken())
 // eslint-disable-next-line camelcase
 __webpack_public_path__ = generateFilePath('mail', '', 'js/')
 
+axios.interceptors.response.use(function(config) {
+	return config
+}, function(error) {
+	if (error.request) {
+		window.OC._processAjaxError(error.request)
```

Comment:
window.OC is a global variable. we've put a lot of effort into getting away from this coding style. I would strongly prefer a solution that works without the presence of a global variable

Site note: the `_` prefix is a convention for *private* methods. So it's not really expected that `OC._*` is called from the outside

Reply:
Yes, i agree that the using of  `OC._*` is not "allowed". I would search for another solution. I've seen how other apps handle it. It seems that this function will be called from server. That is because i wanted to use it too.

Reply:
We could implement it globally in [@nextcloud/axios](https://github.com/nextcloud/nextcloud-axios). Preferably as a opt-in feature so that we don't break all other apps.

Reply:
See https://github.com/nextcloud/nextcloud-axios/pull/381